.gitignore

```diff
@@ -11,3 +11,4 @@ test/*.js
 ~test/jest.config.js
 !test/tests/**/.eslintrc*
 !test/tests/**/*.log
+.idea/
```

Comment:
I think this change would have been made in a separate issue as we should update the `.gitignore` file with more IDE / editors. 

cc @JoshuaKGoldberg

Reply:
Agreed 😄. @criskrzysiu do you want to send that in a separate PR? Otherwise / if you don't have the time one of us can remove it from here & send it too.

Reply:
I will split it, np :)

Reply:
@JoshuaKGoldberg  @KingDarBoja  reverted  .gitignore